Dfe.PrepareConversions/Dfe.PrepareConversions.Tests/Pages/InvoluntaryProject/SearchSchoolIntegrationTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using AngleSharp.Dom;
+using AngleSharp.Html.Dom;
+using Dfe.PrepareConversions.Data.Models.Establishment;
+using FluentAssertions;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Dfe.PrepareConversions.Tests.Pages.InvoluntaryProject
+{
+	public class SearchSchoolIntegrationTests : BaseIntegrationTests
+	{
+		public SearchSchoolIntegrationTests(IntegrationTestingWebApplicationFactory factory) : base(factory)
+		{
+		}
+
+		[Fact]
+		public async Task Should_link_to_school_search()
+		{
+			await OpenUrlAsync($"/project-list");
+			await NavigateAsync("Start a new involuntary conversion project");
+
+			Document.QuerySelector<IHtmlElement>("h1")!.Text().Trim().Should()
```

Comment:
In general, going forwards, do we want this kind of thing (using AngleSharp) to be tested in Cypress? Seems like a little bit of duplication of effort. Should our integration tests just ensure we are getting everything correctly from the API, ready to serve to the front end?

Reply:
Maybe, I think you could argue either way. Perhaps we should all have that discussion together, lets wait till Tim's back.
In the meantime I can leave it in or take it out, I don't feel particularly strongly either way. What's you preference?

Reply:
Keep it in :)